src/components/item/item.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from "classnames";
+import React, { PureComponent, ReactNode } from "react";
+import "../../styles/components/_item.scss";
+import Icon from "../icon";
+
+class Item extends PureComponent<IItemProps> {
+  private static defaultProps = {
```

Comment:
bunu 

 const {
      children,
      className,
      icon,
      dotted = true, <-----
      ...props
    } = this.props;

yapinca oluyor